src/components/ec-submenu/ec-submenu.spec.js

```diff
@@ -0,0 +1,76 @@
+import { mount } from '@vue/test-utils';
+import EcSubmenu from './ec-submenu.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+function mountSubmenu(props, mountOpts) {
+  return mount(EcSubmenu, {
+    propsData: { ...props },
+    stubs: ['router-link'],
+    ...mountOpts,
+  });
+}
+
+const submenu = [
+  {
+    headerTitle: 'Submitted Requests',
+    href: String,
+    additionalText: '(30)',
+    slotName: 'menu-item-1-content',
+  },
+  {
+    headerTitle: 'Repayments',
+    route: '/submitted',
+    additionalText: '(30)',
+    slotName: 'menu-item-2-content',
+  },
+  {
+    headerTitle: 'Another Request',
+    route: '/submitted',
+    additionalText: '(30)',
+    slotName: 'menu-item-3-content',
+  },
+];
+
+const slots = {
+  'menu-item-1-content': '<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Temporibus, tempora esse? Fugit minus consequatur harum culpa sed laborum nulla expedita molestias, quia cupiditate commodi repudiandae labore dolore dolor. Eum sapiente repellendus tempora tenetur accusantium, molestias hic adipisci aspernatur, sed possimus explicabo culpa quas laudantium? Quidem sunt totam maxime magni minus dignissimos eos libero nobis accusantium aspernatur qui inventore odio adipisci at dolorum consectetur velit ipsa quia assumenda porro, consequatur nisi mollitia deserunt. Iusto possimus debitis assumenda harum atque libero voluptates fugit quibusdam ipsa voluptatibus eligendi ex doloremque ducimus molestias natus adipisci, earum facere repudiandae sint nobis error maiores iste. Quam.</p>',
```

Comment:
the 3 slots have the same content, can we extract the lorem ipsum to a constant? Actually, you could easily refactor this further and make this a function